web/config/sync/core.extension.yml

```diff
@@ -11,6 +12,8 @@ module:
   link: 0
   node: 0
   page_cache: 0
+  page_manager: 0
+  page_manager_ui: 0
```

Comment:
Should we have this enabled as a part of the default config?

I am thinking that it is only needed in development scenarios.